src/goto-programs/show_goto_functions.h

```diff
@@ -15,6 +15,12 @@ class goto_functionst;
 class namespacet;
 class goto_modelt;
 
+#define GOTO_SHOW_FUNCTIONS_OPTIONS \
+  "(show-goto-functions)"
+
+#define GOTO_SHOW_FUNCTIONS_HELP \
+  " --show-goto-functions         show goto program\n"
+
```

Comment:
I've replaced yours with `OPT_GOTO_CHECK` but I'm not certain totally sold on the name, specifically I'm not sure the word `GOTO` is adding any meaning to the variable (though I think `OPT_CHECK` implies some kind of checking of options), maybe `OPT_CHECKS` or `OPT_ENABLE_CHECKS`?